lib/test/test-8/test-8.test.js

```diff
@@ -2037,19 +2037,6 @@ describe('Populate class values (valid & invalid)', () => {
     }).toThrow();
   });
 
-  test('set vulnerabilityId', ()=> {
```

Comment:
Not entirely related to this change but it's referring to vulnerabilityId.
In risk-attack-path.js can we delete line 43-44 as it is duplicated?